src/app/NavBar.tsx

```diff
@@ -0,0 +1,130 @@
+import React from "react";
+import Link from "next/link";
+import Image from "next/image";
+
+import logo from "src/app/assets/logos/Cobuild_Logo.svg";
+
+const Navbar: React.FC = () => {
+  return (
+    <nav className="navbar bg-base-100">
+      <div className="navbar-start">
+        <BurgerMenu />
+        <Logo />
+      </div>
+      <div className="navbar-center hidden md:flex">
+        <NavigationLinks />
+      </div>
+      <div className="navbar-end">
+        <UserProfile />
+      </div>
+    </nav>
+  );
+};
+
+const BurgerMenu: React.FC = () => {
```

Comment:
One other tiny thing! Can we break up components into separate files please. In this case they're all related to the navbar, so I don't mind if they all live in the same folder, eg `components/Navbar` or similar.